examples/playlist_all_tracks.py

```diff
@@ -0,0 +1,23 @@
+# get all non-local tracks of a playlist
+from spotipy.oauth2 import SpotifyClientCredentials
+import spotipy
+from pprint import pprint as pp
+
+sp = spotipy.Spotify(client_credentials_manager=SpotifyClientCredentials())
```

Comment:
afaik removing tracks from playlists needs user authentication, did you try this code?